index.js

```diff
@@ -18,6 +18,17 @@ const internals = {
     })
 };
 
+const _extractVersionFromPath = function (request, options) {
+    
+    const apiVersionSegment = request.path.split('/')[2]
```

Comment:
There’s nothing that says the version must be located at position 2 in the path. I’ve written servers that have it at position 1 and position 2.

Reply:
What would be your proposed solution? This solution assumes that the routes are prefixed with `/api/`.

Maybe a regex that picks the first match for `/^v[0-9]+$/` and extracts the numeric value from that string?

Reply:
Something like this maybe.

```js
let path = '/api/v1/blocks'
// let path = '/v1/blocks'

if ((/(v[0-9]+)/).test(path)) {
  return path.match(/(v[0-9]+)/)[0].substr(1)
}

return null
```